specification/communication/data-plane/PhoneNumbers/preview/2023-10-01-preview/phonenumbers.json

```diff
@@ -1467,6 +1864,270 @@
         }
       }
     },
+    "PhoneNumbersBrowseRequest": {
+      "required": [
+        "phoneNumberType"
+      ],
+      "type": "object",
+      "properties": {
+        "phoneNumberType": {
```

Comment:
We recommend "kind" over "type" for discriminators. "Type" can cause conflicts and confusion in various SDK languages.

Reply:
This is reusing a GA'd definition.